functions.php

```diff
@@ -88,6 +91,119 @@ function add_to_twig( $twig ) {
 		return $twig;
 	}
 
+	/**
+	 * Adds the individual sections, settings, and controls to the theme customizer
+	 */
+	function cdtheme_customizer( $wp_customize ) {
+		$this->set_postal_address_section($wp_customize);
+		$this->set_social_accounts_section($wp_customize);
+
+	}
+
+	function set_postal_address_section($wp_customize) {
+		$fields = [
+			[
+				'label' => 'Addressee',
+				'default' => 'CoderDojo Foundation,',
+				'slug' => 'postal_addressee',
+			],
+			[
+				'label' => 'Line one',
+				'default' => 'Dogpatch Labs Unit 1,',
+				'slug' => 'postal_line1',
+			],
+			[
+				'label' => 'Line two',
+				'default' => 'The CHQ building,',
+				'slug' => 'postal_line2',
+			],
+			[
+				'label' => 'Line three',
+				'default' => 'Custom House Quay,',
+				'slug' => 'postal_line3',
+			],
+			[
+				'label' => 'Line four',
+				'default' => 'Dublin 1, Ireland',
+				'slug' => 'postal_line4',
+			],
+		];
+		$wp_customize->add_section(
+			'postal_address',
+				[
+					'title' => 'Postal address',
+					'description' => 'Set your postal address',
+					'priority' => 35,
+				]
+		);
+
+		foreach($fields as $field) {
+			$wp_customize->add_setting(
+					$field['slug'], ['default' => $field['default']]
+			);
+
+			$wp_customize->add_control(
+				$field['slug'],
+				[
+					'label' => $field['label'],
+					'section' => 'postal_address',
+					'type' => 'text',
+				]
+			);
+		}
+	}
+
+	function set_social_accounts_section($wp_customize) {
+		$accounts = [
```

Comment:
this could be exported to pods as 'tag' (fb, twitter..), 'baseUrl', 'username', 'icon'